server/repositories/categoryFeaturedContent.js

```diff
@@ -6,7 +6,7 @@ const config = require('../config');
 const { featuredContentResponseFrom } = require('../utils/adapters');
 
 function categoryFeaturedContentRepository(httpClient) {
-  async function contentFor({ categoryId, establishmentId, number = 8 } = {}) {
+  async function contentFor({ categoryId, establishmentId, number = 10 } = {}) {
```

Comment:
@andrewrlee the limit for category featured content can be set here. Set to 10 for now as outlined in the Trello card